src/middleware/jsonld.ts

```diff
@@ -1,17 +1,31 @@
+import ParserJsonld from '@rdfjs/parser-jsonld';
+import SerializerJsonld from '@rdfjs/serializer-jsonld-ext';
```

Comment:
These are both streaming, which is a bit redundant given the dataset isn't. Not found a parser/serializer that isn't streaming yet.

Reply:
The streaming is both about the input, which dataset maybe is not, bu also about streaming the output content ;)

Reply:
Yeah, mean in terms out parser output and serialiser input!